packages/jaeger-ui/src/components/TracePage/TracePageHeader/TracePageSearchBar.tsx

```diff
@@ -80,20 +80,21 @@ export function TracePageSearchBarFn(props: TracePageSearchBarProps & { forwarde
 
   return (
     <div className="TracePageSearchBar">
-      <Tooltip
-        arrowPointAtCenter
-        mouseLeaveDelay={0.5}
-        placement="bottom"
-        overlayStyle={{ maxWidth: '600px' }} // This is a large tooltip and the default is too narrow.
-        title={renderTooltip()}
-      >
         {/* style inline because compact overwrites the display */}
         <Input.Group className="ub-justify-end" compact style={{ display: 'flex' }}>
           <UiFindInput
             inputProps={uiFindInputInputProps}
             forwardedRef={forwardedRef}
             trackFindFunction={trackFilter}
           />
+        <Tooltip
+        arrowPointAtCenter
+        placement="bottomLeft"
+        trigger='click'
+        overlayStyle={{ maxWidth: '600px' }} // This is a large tooltip and the default is too narrow.
+        title={renderTooltip()}>
+        <Button htmlType="button"><IoHelp className="SearchForm--hintTrigger" /></Button>
+        </Tooltip>
           {navigable && (
             <>
               <Button
```

Comment:
it would be nice to have a mouseover tooltip on this button, it's not very intuitive what it's supposed to do. The up/down arrows are more obvious, as well as `[x]` button.

Reply:
ok, thank you.

Reply:
![Screenshot 2023-08-16 140219](https://github.com/jaegertracing/jaeger-ui/assets/81948346/909b23de-c6a9-46b5-aa79-666a1f03e2a3)
@yurishkuro fixed.